### ISSUE TITLE
0% documentation Coverage with 0 undocumented symbols

### ISSUE THREAD

**Issue by amitkumar693** [realm/jazzy#1174]
I am getting 0% Documentation every-time:

Checking xcodebuild -showBuildSettings
Assuming New Build System is used.
0% documentation coverage with 0 undocumented symbols
building site
building search index
jam out ♪♫ to your fresh new docs in `docs`

for **jazzy --min-acl**:

/Library/Ruby/Gems/2.3.0/gems/jazzy-0.10.0/lib/jazzy/source_declaration/access_control_level.rb:64:in `from_human_string': cannot initialize AccessControlLevel with '' (RuntimeError)
	from /Library/Ruby/Gems/2.3.0/gems/jazzy-0.10.0/lib/jazzy/config.rb:301:in `block in <class:Config>'
	from /Library/Ruby/Gems/2.3.0/gems/jazzy-0.10.0/lib/jazzy/config.rb:35:in `instance_exec'
	from /Library/Ruby/Gems/2.3.0/gems/jazzy-0.10.0/lib/jazzy/config.rb:35:in `set'
	from /Library/Ruby/Gems/2.3.0/gems/jazzy-0.10.0/lib/jazzy/config.rb:54:in `block in attach_to_option_parser'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/optparse.rb:1547:in `block in parse_in_order'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/optparse.rb:1534:in `catch'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/optparse.rb:1534:in `parse_in_order'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/optparse.rb:1528:in `order!'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/optparse.rb:1620:in `permute!'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/optparse.rb:1642:in `parse!'
	from /Library/Ruby/Gems/2.3.0/gems/jazzy-0.10.0/lib/jazzy/config.rb:432:in `parse_command_line'
	from /Library/Ruby/Gems/2.3.0/gems/jazzy-0.10.0/lib/jazzy/config.rb:389:in `parse!'
	from /Library/Ruby/Gems/2.3.0/gems/jazzy-0.10.0/bin/jazzy:15:in `<top (required)>'
	from /usr/local/bin/jazzy:22:in `load'
	from /usr/local/bin/jazzy:22:in `<main>'

Can someone please help.

**Xcode**: 11.3
**macOS**: Mojave (10.14.6)
**jazzy (0.10.0, 0.9.6)** 

[SwiftDocSample-master.zip](https://github.com/realm/jazzy/files/4473153/SwiftDocSample-master.zip)

**Comment by johnfairh** [comment]
See the help, the syntax would be `jazzy --min-acl private`.

The second problem is that the attached project does not build:
```
; xcodebuild
note: Using new build system
note: Planning build
note: Constructing build description
error: Signing for "DocSample" requires a development team. Select a development team in the Signing & Capabilities editor. (in target 'DocSample' from project 'DocSample')
```

Jazzy is not telling you this because of a bug in a dependency fixed somewhere between 0.10.0 and latest.

**Comment by amitkumar693** [comment]
After changing the syntax to : `jazzy --min-acl private`

xcodebuild[8587:1460883]  DTDeviceKit: deviceType from 00008006-000F05593A07002E was NULL
Running xcodebuild
Checking xcodebuild -showBuildSettings
Assuming New Build System is used.
0% documentation coverage with 0 undocumented symbols
building site
building search index
jam out ♪♫ to your fresh new docs in `docs`

**Comment by johnfairh** [comment]
What about fixing the second problem?

**Comment by amitkumar693** [comment]
@johnfairh :  I added my apple credentials in that, then only I tried `jazzy --min-acl private`. 

I have also added the sample code as zip file. If possible can you try on your system to see if it's working ?

**Comment by johnfairh** [comment]
You should run `xcodebuild` and fix the errors it reports.  I already pasted the output of running `xcodebuild` on my system against the zip file -- the contents don't seem to have changed since then.

**Comment by amitkumar693** [comment]
Really appreciate your help @johnfairh.

I don't know what was the issue. 

I started from the scratch as follows:

1. ` sudo gem install jazzy` which gave me an error 
      > While executing gem ... (Gem::FilePermissionError)
      > You don't have write permissions for the /usr/bin directory. 
2. Then I tried : `sudo gem install -n /usr/local/bin jazzy` with help of this URL : https://stackoverflow.com/questions/2893889/how-do-i-fix-the-you-dont-have-write-permissions-into-the-usr-bin-directory
3. And then `jazzy --min-acl private` gave me the output:
  Running xcodebuild
Checking xcodebuild -showBuildSettings
Assuming New Build System is used.
Parsing AppDelegate.swift (1/2)
Parsing ViewController.swift (2/2)
47% documentation coverage with 10 undocumented symbols
included 19 private, fileprivate, internal, public, or open symbols
building site
building search index
jam out ♪♫ to your fresh new docs in `docs`
